test/lld/duplicate_imports.wat.out

```diff
@@ -32,6 +36,20 @@
  (func $__wasm_call_ctors
   (nop)
  )
+ (func $dynCall_fdd (param $fptr i32) (param $0 f64) (param $1 f64) (result f32)
+  (call_indirect (type $f64_f64_=>_f32)
+   (local.get $0)
+   (local.get $1)
+   (local.get $fptr)
+  )
+ )
+ (func $dynCall_ffd (param $fptr i32) (param $0 f32) (param $1 f64) (result f32)
+  (call_indirect (type $f32_f64_=>_f32)
+   (local.get $0)
+   (local.get $1)
+   (local.get $fptr)
+  )
+ )
```

Comment:
These dynCalls are generated because the input file contains `invoke_` functions. Before they were not targets for dynCall generations because `EmscriptenGlueGenerator` looked for the format generated by the LLVM backend, like `__invoke_void_i32`, and not `invoke_vi`.